composer.json

```diff
@@ -34,7 +34,7 @@
         "propel/propel1": "~1.7",
         "symfony/yaml": "2.*",
         "symfony/translation": "~2.0",
-        "symfony/validator": "~2.0",
+        "symfony/validator": "~2.0, <2.2",
```

Comment:
This is not acceptable as we have to support ``symfony/symfony`` 2.7 for example. Better to remove the support of sf < 2.3